curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/exercise-tracker.md

```diff
@@ -94,7 +98,7 @@ tests:
       }
     }
     "
-  - text: I can retrieve a full exercise log of any user by getting /api/exercise/log with a parameter of userId(_id). App will return the user object with added array log and count (total exercise count).
+  - text: You can make a `GET` request to `/api/exercise/log` with a parameter of `userId=_id` to retrieve a full exercise log of any user. The returned response will be the user object with a `log` array of all the exercises added. Each log item has the `description`, `duration`, and `date` properties. Also included is a `count` key with the the total exercise count as its value.
```

Comment:
Is there any reason the mention of the `count` key sounds like it is tested twice?